components/calendar/src/dangi.rs

```diff
@@ -423,7 +425,11 @@ impl Dangi {
     fn format_dangi_year(year: i32) -> FormattableYear {
         let era = Era(tinystr!(16, "dangi"));
         let number = year;
-        let cyclic = Some(div_rem_euclid(number - 1 + 364, 60).1 + 1);
+        let cyclic = Some(
+            div_rem_euclid(number.saturating_sub(1).saturating_add(364), 60)
+                .1
+                .saturating_add(1),
+        );
```

Comment:
Nit: make it `Some(div_rem_euclid64(number as i64 - 1 + 364, 60).1 as i32 + 1);`

Reply:
Done